app/scenes/Home.tsx

```diff
@@ -76,7 +69,7 @@ function Home() {
             <PaginatedDocumentList
               documents={documents.recentlyUpdated}
               fetch={documents.fetchRecentlyUpdated}
-              empty={<Empty>{t("Weird, this shouldn’t ever be empty")}</Empty>}
+              empty={<Empty>{t("Weird, this shouldn't ever be empty")}</Empty>}
```

Comment:
@codegen update the PR to restore the original translation strings in this file – they don't need changing.